status-im/status-react
# [ISSUE #2177] Moved component into ui

addresses #2177

### Summary:

Migrate ns `components` to `ui.components`.

### Steps to test:

No fix nor new features: app needs to be sanity-checked.

- Open Status
- Make sure there are no UI regression

status: ready
